projects/questionset-editor-library/src/lib/services/tree/tree.service.ts

```diff
@@ -253,35 +253,40 @@ export class TreeService {
   }
 
   updateEvaluable(nodeId){
```

Comment:
The method call of `updateEvaluable` in this service can you call it based on some condition check

Reply:
Inside the method we have condition based update for evalMode. If I keep outside this will requires more changes and testing now.
@rajnishdargan